components/Chips/src/MDCChipCollectionViewCell.m

```diff
@@ -71,7 +74,11 @@ - (CGSize)intrinsicContentSize {
 }
 
 - (void)setSelected:(BOOL)selected {
-  [super setSelected:selected];
+  // Skipping state management in the current cell. States is managed in self.chipView instead.
+  if (!self.chipView.enableRippleBehavior) {
+    [super setSelected:selected];
```

Comment:
How will this work if the `enableRippleBehavior` flag is toggled on a selected collection view cell? I would expect to see KVO of the Chip's property so that the superclass can be updated. For example, if a selected cell enables Ripple, then tell the superclass that it's no longer selected so the property can be managed by Ripple.